lib/opensusebasetest.pm

```diff
@@ -29,6 +30,53 @@ sub export_logs {
     select_console 'root-console';
     save_screenshot;
 
+    # check if wicked is installed and turn on debug, get all needed wicked logs if network interface is down
+    if (script_run 'rpm -q wicked') {
```

Comment:
Maybe it's easier to read, but we will always write this condition instead of just export_logs; ?
It's exporting wicked logs only when wicked interface is down and setup network interface to get logs at all, otherwise it's default export_logs as before.